geonode/static/bower.json

```diff
@@ -27,6 +27,9 @@
     "angular-leaflet-directive": "0.7.9",
     "zeroclipboard": "2.1.5",
     "leaflet-fullscreen": "https://github.com/Leaflet/Leaflet.fullscreen.git",
+    "leaflet-opacity": "https://github.com/lizardtechblog/Leaflet.OpacityControls.git#master",
+    "leaflet-navbar": "1.0.1",
+    "leaflet-measure": "2.1.7",
```

Comment:
Added leaflet plugins reference in bower.
This will install the plugin when doing `bower install`